package.json

```diff
@@ -28,16 +28,17 @@
   "author": "Simon Tsvilik",
   "license": "MIT",
   "devDependencies": {
-    "babel": "^6.23.0",
-    "babel-cli": "^6.26.0",
+    "@babel/preset-env": "^7.4.3",
+    "@babel/cli": "^7.4.3",
+    "@babel/core": "^7.4.3",
     "babel-preset-env": "^1.7.0",
```

Comment:
I believe having two versions of babel will cause conflicts

Reply:
@stsvilik I had problems with the tests on `master` as well, trying to debug a failure in Promises. Do they pass for you?

Reply:
I have fixed that, thanks for catching it